tests/common/test_event.py

```diff
@@ -28,7 +28,8 @@
     WALAEventOperation, parse_xml_event, parse_json_event, AGENT_EVENT_FILE_EXTENSION, EVENTS_DIRECTORY
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.protocol.goal_state import GoalState
-from tests.protocol import mockwiredata, mock_wire_protocol
+from tests.protocol import mockwiredata
+from tests.protocol.mocks import mock_wire_protocol
```

Comment:
I moved mock_wire_protocol to a different file (mocks.py)

Reply:
I also changed the usage from

` with mock_wire_protocol.create(...)... 
`

to

` with mock_wire_protocol(...)... 
`

As I'm adding more functionality to the mocks the latter feels better.

Reply:
Yup - Looks cleaner. Thanks